asv.conf.json

```diff
@@ -0,0 +1,21 @@
+
+{
+    "version": 1,
+    "project": "stardis",
+    "project_url": "https://tardis-sn.github.io/tardis",
```

Comment:
this needs to be changed

Reply:
I'm not sure if there is a separate project_url for stardis yet